tests/interop/main.rs

```diff
@@ -87,7 +87,27 @@ fn fuchsia_go_tuf_transition_m4_consistent_snapshot_true() {
             .join("interop")
             .join("fuchsia-go-tuf-transition-M4")
             .join("consistent-snapshot-true"),
-    )
+    );
+}
+
+// Tests to catch changes to the way we generate metadata.
+#[test]
+fn fuchsia_rust_tuf_identity_consistent_snapshot_false() {
```

Comment:
As far as I can tell; these are for rust-tuf in general, so no need for the word "fuchsia" here. (for the go-tuf metadata, the metadata is from the fuchsia fork of go-tuf, hence the name)

Reply:
Removed the "fuchsia"